src/sql/workbench/services/connection/browser/cmsConnectionWidget.ts

```diff
@@ -53,8 +54,14 @@ export class CmsConnectionWidget extends ConnectionWidget {
 		super(options, callbacks, providerName, _themeService, _contextViewService, _connectionManagementService, _capabilitiesService,
 			_clipboardService, _configurationService, _accountManagementService);
 		let authTypeOption = this._optionsMaps[ConnectionOptionSpecialType.authType];
-		authTypeOption.defaultValue = this.getAuthTypeDisplayName(AuthenticationType.Integrated);
-		this._authTypeSelectBox = new SelectBox(authTypeOption.categoryValues.map(c => c.displayName), authTypeOption.defaultValue, this._contextViewService, undefined, { ariaLabel: authTypeOption.displayName });
+		if (authTypeOption) {
+			if (OS === OperatingSystem.Windows || authTypeChanged) {
```

Comment:
>authTypeChanged [](start = 41, length = 15)

Why does authTypeChanged control this? If we're not on a windows machine why would we ever want to set the default value to Integrated?

Reply:
Because this feature only works with Integrated Auth